README.md

```diff
@@ -157,9 +163,10 @@ Example
 See [CRYSTAL_API.md](./CRYSTAL_API.md)
 
 ## Why?
-(1) It is tiring to have to stop and restart an app on every change.
 
-(2) Docker!
+* 1 It is tiring to have to stop and restart an app on every change.
```

Comment:
Since this is adding bullets, I would just remove the numbers altogether. Otherwise i don't think this change is necessary.  
So I'd say we either remove the numbers or keep as it was.